packages/api/schema.json

```diff
@@ -20,8 +20,14 @@
   "definitions": {
     "api": {
       "type": "object",
-      "required": ["identifier", "installerVersion", "integrity"],
+      "required": ["createdAt", "identifier", "installerVersion", "integrity"],
```

Comment:
this would make this a breaking change, right?

Reply:
hm yeah good point. i'll bump the lockfile to v2 and for lockfile v1 users if it's missing i'll not print anything.

Reply:
talked offline and we're dropping `version` instead from the lockfile and are relying just on the versioned `$schema`.